### ISSUE TITLE
EC2Metadata missing from AWSSDK.EC2 3.3.2

### ISSUE THREAD

**Issue by aggieben** [aws/aws-sdk-net#481]
I found this comment on another issue (#339) that has already been closed:

https://github.com/aws/aws-sdk-net/issues/339#issuecomment-211608590

> EC2Metadata will probably be out for awhile because it is based on System.Net.HttpWebRequest which isn't available CoreCLR. We have a few utility classes like this in the SDK that uses the HttpWebRequest directly so we had to remove them out of the CoreCLR version. We need to work on a common abstraction between System.Net.HttpWebRequest and System.Net.Http.HttpClient that our utility classes can use and then we can add back the utility classes.

Will this be fixed soon?  This comment was from April.

**Comment by socketnorm** [comment]
We found that having EC2Metadata made more sense in the Core package so we replicated it in Core as Amazon.Util.EC2InstanceMetadata. We kept EC2Metadata for .NET Framework for backward compatibility but didn't put in .NET Core because it relied on classes not available in .NET Core.

I think what we need to do is add the obsolete attribute on EC2Metadata pointing users to the newer and preferred Amazon.Util.EC2InstanceMetadata.
